lib/rack_middleware/robots_generator.rb

```diff
@@ -5,13 +5,16 @@ class RobotsGenerator
   # Use the config/robots.txt in production.
   # Disallow everything for all other environments.
   def self.call(env)
+    return [404, {}, []] if env[:current_marketplace].nil?
```

Comment:
Isn't 404 more accurate information? "This page does not exists" vs "This page does exist but I don't want you to index it"